unlock-app/src/__tests__/actions/user.test.ts

```diff
@@ -116,4 +118,17 @@ describe('user account actions', () => {
       expect(changePassword(newPassword)).toEqual(expectedAction)
     })
   })
+
+  describe('user authentication actions', () => {
+    it('should create an action to deliver a requested password', () => {
+      expect.assertions(1)
+      const password = 'guest'
+      const expectedAction = {
+        type: GOT_PASSWORD,
```

Comment:
I'm not sure this is the best action name. PASSWORD_ENTERED?

Reply:
Let's not quibble over semantically identical action names